community/server/pom.xml

```diff
@@ -159,7 +159,7 @@
 
     <dependency>
       <groupId>ch.qos.logback</groupId>
-      <artifactId>logback-classic</artifactId>
+      <artifactId>logback-core</artifactId>
```

Comment:
@tinwelint It's because of HTTP logging in Jetty :/. I think we'd be able to write our own thing for that as well, if we wanted to get rid of logback altogether. I def. understand the reasons for wanting to not expose logback config to users, but I still haven't understood why we want to remove logback itself.

@cleishm yep